vital/types/detected_object.cxx

```diff
@@ -62,7 +78,7 @@ ::clone() const
   }
 
   auto new_obj = std::make_shared<kwiver::vital::detected_object>(
-    *this->m_bounding_box, this->m_confidence, new_type );
```

Comment:
(**CRITICAL**) Doesn't this need to copy the location also? And by "copy", I mean "clone", especially as IIRC `const geo_point` is mutable.